engine/.github/workflows/mega-linter.yml

```diff
@@ -0,0 +1,82 @@
+---
```

Comment:
Note the acceptance criteria states explicitly that this user story should not include CI/CD, that is covered by #12

Reply:
In fact I don't think having a non-root .GitHub folder will work....